pyproject.toml

```diff
@@ -0,0 +1,34 @@
+# common tooling configs
+[tool.black]
```

Comment:
Curious about why this was replicated here in addition to `client/python/cryoet_data_portal`? 😅

Reply:
`pre-commit` searches for the tooling settings under the root directory; also this means we can have one configuration for both the backend and client folders (which can be overridden by their separate `pyproject.toml`s